jina/jaml/parsers/__init__.py

```diff
@@ -22,8 +22,6 @@ def _get_all_parser(cls: Type['JAMLCompatible']):
         return _get_exec_parser()
     elif issubclass(cls, BaseGateway):
         return _get_gateway_parser()
-    else:
```

Comment:
else, raise error.
Ideally, each JAML compatible class should select its parser rather than having to add code to select it (but not so important for now)